src/main/java/edu/harvard/iq/dataverse/export/ExportService.java

```diff
@@ -209,17 +205,17 @@ public void exportFormat(Dataset dataset, String formatName) throws ExportExcept
                     if (releasedVersion == null) {
                         throw new IllegalStateException("No Released Version");
                     }
-                    JsonPrinter jsonPrinter = new JsonPrinter(settingsService);
-                    final JsonObjectBuilder datasetAsJsonBuilder = jsonPrinter.jsonAsDatasetDto(releasedVersion);
+                    final JsonObjectBuilder datasetAsJsonBuilder = JsonPrinter.jsonAsDatasetDto(releasedVersion);
```

Comment:
I'm wondering if no longer passing "settingsService" into the JsonPrinter constructor has led to #5185 about ":ExcludeEmailFromExport" no longer working.